java/src/main/java/ai/rapids/cudf/Cuda.java

```diff
@@ -269,18 +269,30 @@ public void close() {
    * @param kind  - Type of transfer. {@link CudaMemcpyKind}
    */
   static void memcpy(long dst, long src, long count, CudaMemcpyKind kind) {
-    memcpy(dst, src, count, kind.getValue());
+    memcpy(dst, src, count, kind, DEFAULT_STREAM);
   }
 
-  private static native void memcpy(long dst, long src, long count, int kind) throws CudaException;
+  /**
+   * Copies count bytes from the memory area pointed to by src to the memory area pointed to by
+   * dst.
+   * Calling cudaMemcpy() with dst and src pointers that do not
```

Comment:
The docs implies this is a synchronous copy and refer to `cudaMemcpy` but it is the async version.  Docs should call out the asynchronous behavior.

Reply:
Done.